src/integration.test.ts

```diff
@@ -230,7 +230,7 @@ describe(`Integration test for portal ${portal}`, () => {
 
     it("Should get plaintext file metadata", async () => {
       // TODO: Remove once the metadata changes are in prod.
-      const portal = "https://siasky.dev";
+      const portal = "https://siasky.xyz";
```

Comment:
I'm not sure which metadata changes are meant here. Are they still not on prod?

Pointing tests to xyz is a little risky - this is the server we restart and redeploy most often. If you're sure about it then I don't mind, of course. I'm just wondering if there is a better solution.